### ISSUE TITLE
Run on GPU

### ISSUE THREAD

**Issue by chiragagarwal1999** [NumbersStationAI/NSQL#14]
Can this model run on GPU? If yes, how do i connect my GPU to it

**Comment by senwu** [comment]
Yes, if you use transformers package here is one note: https://huggingface.co/docs/transformers/perf_infer_gpu_one
